templates/stevenage/foot.html

```diff
@@ -15,33 +15,46 @@
 	<div class="secondaryFooterWrapper">
 		<div id="secondaryFooter" class="group">
 			   		
-			<form action="http://www.govmetric.com/WebMetric/websatisfaction.aspx" method="post">
-				<div class="ratePage">
-					<input type="hidden" name="organisationid" value="343" />
-					<input type="image" src="http://www.govmetric.com/images/green-face-30x30.gif" alt="Rate as Good" name="metric1" value="1" />
-					<input type="image" src="http://www.govmetric.com/images/amber-face-30x30.gif" alt="Rate as Average" name="metric2" value="2" />
-					<input type="image" src="http://www.govmetric.com/images/red-face-30x30.gif" alt="Rate as Poor" name="metric3" value="3" />
-					<p class="inst"><img src="http://www.stevenage.gov.uk/Images/rateThisPage.gif" alt="Please rate this page." /></p>
-				</div>
+<!-- WebMetric snippet starts here -->
+
+<div class="gm_sidebar">
+    <a
+        href="https://websurveys.govmetric.com/theme/gm/1340?Q_RATINGID=3"
+        target="_blank"
+        class="gm_sidebar_anchor"
+        title="rate this page as GOOD"
+        rel="nofollow">
+        <img style="width:30px;height:30px;" src="https://wsstatic.govmetric.com/imgs/smileys/large/good.png" alt="green smiley (good)" />
+    </a>
+    <a
+        href="https://websurveys.govmetric.com/theme/gm/1340?Q_RATINGID=2"
+        target="_blank"
+        class="gm_sidebar_anchor"
+        title="rate this page as AVERAGE"
+        rel="nofollow">
+        <img style="width:30px;height:30px;" src="https://wsstatic.govmetric.com/imgs/smileys/large/average.png" alt="orange smiley (average)" />
+    </a>
+    <a
+        href="https://websurveys.govmetric.com/theme/gm/1340?Q_RATINGID=4"
+        target="_blank"
+        class="gm_sidebar_anchor"
+        title="rate this page as POOR"
+        rel="nofollow">
+        <img style="width:30px;height:30px;" src="https://wsstatic.govmetric.com/imgs/smileys/large/poor.png" alt="red smiley (poor)" />
+    </a>
+
+                <p class="inst"><img src="/Images/rateThisPage.gif" alt="Please rate this page." />
```

Comment:
This image doesn't exist locally, it probably wants the `http://www.stevenage.gov.uk/` prefix.